### PR TITLE
Solution)

### PR DESCRIPTION
**Pull request by Gr1zzZzzly**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```
Comment (mltdll) [inline app/main.py:5]: I think you should introduce separate variables for both arguments. It's hard to follow your code's logic when they are just `command[1]` and `command[2]`.

**Review by mltdll — CHANGES_REQUESTED** [review]

**Review by MasterpieceElbow — APPROVED** [review]
GJ!
